src/audiomixerboard.cpp

```diff
@@ -1305,6 +1307,49 @@ void CAudioMixerBoard::SetAllFaderLevelsToNewClientLevel()
     }
 }
 
+void CAudioMixerBoard::AutoAdjustAllFaderLevels()
+{
+    QMutexLocker locker ( &Mutex );
+
+    for ( int i = 0; i < MAX_NUM_CHANNELS; ++i )
+    {
+        // only apply to visible faders (and not to my own channel fader)
+        if ( vecpChanFader[i]->IsVisible() && ( true /* i != iMyChannelID */ ) )
+        {
+            // get averaged level meter value (range 0...8)
+            float levelMeter = vecAvgLevels[i];
+
+            // map meter output level to deciber (range -50dB...0dB)
```

Comment:
Is the "-50dB" the `targetdBLevel`?  Or the -40f or the 35f?  Ideally name all of these values then use the names.